packages/storage-plugin/src/storage.plugin.ts

```diff
@@ -29,12 +28,11 @@ declare const ngDevMode: boolean;
 export class NgxsStoragePlugin implements NgxsPlugin {
   constructor(
     @Inject(NGXS_STORAGE_PLUGIN_OPTIONS) private _options: NgxsStoragePluginOptions,
-    @Inject(STORAGE_ENGINE) private _engine: StorageEngine,
-    @Inject(PLATFORM_ID) private _platformId: string
+    @Inject(STORAGE_ENGINE) private _engine: StorageEngine
   ) {}
 
   handle(state: any, event: any, next: NgxsNextPluginFn) {
-    if (isPlatformServer(this._platformId) && this._engine === null) {
+    if (this._engine === null) {
```

Comment:
I am not super confident on this one.
I checked the intial PR and its been said if is SSR and custom store. But on SSR the engine will default to null, and if is custom its not null anymore.